src/components/DatasetCreateView.jsx

```diff
@@ -186,7 +186,7 @@ export class DatasetCreateView extends React.PureComponent {
                 <ManageUsers
                   {...props}
                   addReader={newEmail => this.addReader(newEmail)}
-                  defaultValue="Custodian Email Address"
+                  defaultValue="Add viewer email address"
```

Comment:
Let's have this be override-able by a prop that's passed in and defaults to "Add email address" if nothing is provided.